src/main/target/AIKONF4/target.h

```diff
@@ -0,0 +1,143 @@
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define GYRO_MPU6000_ALIGN       CW0_DEG
+#define ACC_MPU6000_ALIGN        CW0_DEG
+
+#define USE_BARO
+#define USE_BARO_BMP280
+// @note AIKON uses PB7 as CS for MBP280, but BF only uses CS for SPI
+
+#define USE_OSD
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE    SPI3
+#define MAX7456_SPI_CS_PIN      SPI3_NSS_PIN
+#define MAX7456_SPI_CLK         (SPI_CLOCK_STANDARD) // 10MHz
+#define MAX7456_RESTORE_CLK     (SPI_CLOCK_FAST)
+
+#define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+#define M25P16_CS_PIN           SPI2_NSS_PIN
+#define M25P16_SPI_INSTANCE     SPI2
+
+#define USE_VCP
+#define VBUS_SENSING_PIN        PD2
+
+#define USE_UART1
+#define UART1_RX_PIN            PA10
+#define UART1_AHB1_PERIPHERALS  RCC_AHB1Periph_DMA2
+
+#define USE_UART2
+#define UART2_RX_PIN            PA3
+#define UART2_TX_PIN            PA2
+
+#define USE_UART3
+#define UART3_RX_PIN            PB11
+#define UART3_TX_PIN            PB10
+
+#define USE_UART4
+#define UART4_TX_PIN            PA0
+
+#define USE_SOFTSERIAL1
+#define SOFTSERIAL1_TX_PIN      PA9
+
+#define USE_SOFTSERIAL2
+#define SOFTSERIAL2_RX_PIN      PA1
+
+#define SERIAL_PORT_COUNT       7 // VCP, USART1, USART3,USART4, USART6, SOFT_SERIAL1, SOFT_SERIAL2
+
+#define USE_ESCSERIAL
+//@todo redo
+#define ESCSERIAL_TIMER_TX_PIN  PB6  // (HARDARE=0,PPM)
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define SPI1_NSS_PIN            PA4
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define USE_SPI_DEVICE_2
+#define SPI2_NSS_PIN            PB12
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+#define USE_SPI_DEVICE_3
+#define SPI3_NSS_PIN            PA15
+#define SPI3_SCK_PIN            PC10
+#define SPI3_MISO_PIN           PC11
+#define SPI3_MOSI_PIN           PC12
+
+#define USE_I2C
+// @note is this necessary?
+//#define USE_I2C_PULLUP
+#define USE_I2C_DEVICE_1
+#define I2C_DEVICE              (I2CDEV_1)
+#define I2C1_SCL                PB8
+#define I2C1_SDA                PB9
+
+#define USE_ADC
+#define CURRENT_METER_ADC_PIN   PC1
+#define VBAT_ADC_PIN            PC2
+#define RSSI_ADC_PIN            PC3
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+
+#define SERIALRX_PROVIDER       SERIALRX_SBUS
+#define SERIALRX_UART           SERIAL_PORT_USART1
+
+#define DEFAULT_RX_FEATURE          FEATURE_RX_SERIAL
+#define DEFAULT_FEATURES            ( FEATURE_OSD | FEATURE_SOFTSERIAL  )
+#define USE_SERIAL_4WAY_BLHELI_INTERFACE
+
+#define TARGET_IO_PORTA 0xffff
+#define TARGET_IO_PORTB 0xffff
+#define TARGET_IO_PORTC 0xffff
+#define TARGET_IO_PORTD BIT(2)
+
+#define USABLE_TIMER_CHANNEL_COUNT 16
+#define USED_TIMERS  (  TIM_N(1) | TIM_N(2) | TIM_N(3) | TIM_N(4) | TIM_N(5) | TIM_N(8) | TIM_N(9))
```

Comment:
Spacing.

Reply:
You mean `BIT(2)` not being indented to `16` level?

Reply:
Spacing aka Whitespace :P 
(  TIM_N(1)

Reply:
Yep. Double space before, no space after. But this is just nitpicking.